community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/load_bq.py

```diff
@@ -322,7 +323,13 @@ def main():
     # it will try again next time. If some writes succeed, we don't currently
     # have a way to not submit duplicates next time.
     if jobs:
-        bq_submit(jobs)
+        num_batches = (len(jobs) - 1) // BQ_ROW_BATCH_SIZE + 1
+        print(
+            f"loading {num_batches} batches of BigQuery data in batches of size : {BQ_ROW_BATCH_SIZE}"
+        )
+        for indx in range(0, len(jobs), BQ_ROW_BATCH_SIZE):
+            print(f"loading BigQuery data batch {indx} of {num_batches}")
```

Comment:
Misleading message: "loading BigQuery data batch 15000  of 3"

Reply:
could be written as 
`f"loading BigQuery data batch {indx // BQ_ROW_BATCH_SIZE} of {num_batches}"`

or simply don't use `num_batches`